tests/ray/test_evaluator.py

```diff
@@ -20,12 +21,14 @@ class TestEvaluator(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls) -> None:
-        # RL默认使用FA3
         os.environ["XTUNER_USE_FA3"] = "1"
+        cls.temp_dir = tempfile.TemporaryDirectory()
```

Comment:
Suggest using a different tempdir for different testcases defined （define tempdir in setUp）